source/common/http/conn_manager_impl.h

```diff
@@ -128,10 +130,28 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     Tracing::Span& activeSpan() override;
     Tracing::Config& tracingConfig() override;
 
+    // Functions to set or get iteration state.
+    bool canIterate() { return iteration_state_ == IterationState::Continue; }
+    bool shouldNotIterate() { return iteration_state_ != IterationState::Continue; }
```

Comment:
nit: for a bit less code can we just do `!canIterate()` where needed?

Reply:
Fixed. Thanks!